jekyll/_cci2/hello-world-windows.md

```diff
@@ -0,0 +1,131 @@
+* To request access to Windows Preview using [this form](https://docs.google.com/forms/d/e/1FAIpQLSfspug2MP0eTK8eRC1_FpiDQzNHkk8a36fflN_za29CwGzGoQ/viewform).
+
+
+# Overview of the Windows executor
+
+The Windows build environment (or `executor`) gives users the tools to build Windows projects, such as a Universal Windows Platform (UWP) application, a .NET executable, or Windows-specific (like the .NET framework) The Windows executor uses the Server Core version of Windows Server 2019 DataCenter Edition.
+
+A windows executor will have 4 vCPUs and 15GB of ram.
+
+On the Windows executor, _Git_, the package manager _Chocolatey_, and _7zip_ will be installed. Powershell is the default shell (Bash and Command are available to be manually selected). Other dependencies may be installed later on.
+
+**Limitations**
+
+As the Windows executor is still in a preview phase, there is some functionality that is not yet supported. Currently Windows on CircleCi does not support the following CircleCI features:
+
+*   [Docker Layer Caching]({{site.baseurl}}/2.0/docker-layer-caching)
+*   Docker support
+
+# Example configuration file
+
+The following code snippet is an absolute minimum to get started with Windows on CircleCI:
+
+```yaml
+version: 2.1
+
+orbs:
+  win: sandbox/windows-tools@dev:preview
+
+jobs:
+  build:
+    executor: win/preview-default
+    steps:
+      - checkout
+      - run: echo 'Hello, Windows'
+```
+
+# Example Application
+
+Let’s consider a more advanced (but still introductory) "hello world" application using the Windows executor. This [example application](https://github.com/CircleCI-Public/circleci-demo-windows) still prints "Hello World" to the console, but does so using .NET core to create an executable, uses dependency caching, and creates an artifact on every build.
+
+You can view the entire configuration [here](https://github.com/CircleCI-Public/circleci-demo-windows/blob/master/.circleci/config.yml).
+
+```yaml
+version: 2.1
+```
+
+Above, we start by declaring that we will use version `2.1` of CircleCI, giving us access to [Orbs](https://circleci.com/orbs/) and [Pipelines]({{site.baseurl}}/2.0/build-processing/).
+
+```yaml
+orbs:
+  win: sandbox/windows-tools@dev:preview
+```
+
+Next, we declare orbs that we will be using in our build. We will only use the [windows-tools orb](https://circleci.com/orbs/registry/orb/circleci/windows-tools) to help us get started.
+
+```yaml
+jobs:
+  build:
+    executor:
+      name: win/preview-default
+      shell: powershell.exe
+```
+
+Under the `jobs` key, we set the executor via the orb we are using. We can also declare the default shell to be applied across future steps in the configuration.
+
+```yaml
+    steps:
+      - checkout
+      - run:
+          name: "Install dotnet core" # https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script
+          command: .circleci/dotnet-install.ps1 -InstallDir C:\cli
```

Comment:
Feel free to do it this way, but we've fixed those pesky path issues the were breaking installations from chocolatey from working before. I've a branch on the demo project to reflect these changes. https://github.com/CircleCI-Public/circleci-demo-windows/pull/1

Reply:
Ah, thank you. I was struggling with what to do to get chocolatey + dotnet happy. Thank you!